pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -327,6 +327,24 @@ public void testMultiValueColumnAggregationQuery(boolean useMultiStageQueryEngin
     Assert.assertTrue(jsonNode.get("resultTable").get("rows").get(0).get(0).asDouble() < 17000);
   }
 
+  @Test(dataProvider = "useBothQueryEngines")
+  void testDual(boolean useMultiStageQueryEngine)
```

Comment:
Just to clarify, we don't actually have the concept of a `DUAL` table in Pinot right?

Reply:
queries without a FROM class are _supposed_ to use DUAL table. Its a carry-over term from oracle culture.